### PR TITLE
UIQM-16 Edit quickMarc accessibility

### PR DESCRIPTION
**Pull request by uladislausamets**
## Purpose
https://issues.folio.org/browse/UIQM-16

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js [hunk 1]
```diff
@@ -76,59 +76,84 @@ const QuickMarcEditorRows = ({
...
-                      data-test-move-up-row
-                      data-index={idx}
-                      data-index-to-switch={idx - 1}
-                      icon="arrow-up"
-                      onClick={moveRow}
-                    />
+                    <FormattedMessage id="ui-quick-marc.record.areaLabel.moveUpRow">
+                      {ariaLabel => (
+                        <IconButton
+                          ariaLabel={ariaLabel}
```
Comment (NikitaSedyx) [inline src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js:82]: please add title prop as well

File: src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js [hunk 2]
```diff
@@ -76,59 +76,84 @@ const QuickMarcEditorRows = ({
...
                 {
                   !withMoveUpRowAction && (
-                    <IconButton
-                      data-test-move-up-row
-                      data-index={idx}
-                      data-index-to-switch={idx - 1}
-                      icon="arrow-up"
-                      onClick={moveRow}
-                    />
+                    <FormattedMessage id="ui-quick-marc.record.areaLabel.moveUpRow">
```
Comment (NikitaSedyx) [inline src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js:79]: `ui-quick-marc.record.moveUpRow`

**Comment by NikitaSedyx** [comment]
FixedField is not covered

File: src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js [hunk 3]
```diff
@@ -36,15 +36,21 @@ const renderSubField = (name, config) => {
...
-                      className={styles.fixedFieldSubFieldByte}
-                      hasClearIcon={false}
-                      data-testid={`fixed-field-${config.type}`}
-                    />
+                    <FormattedMessage id="ui-quick-marc.record.subfield">
+                      {ariaLabel => (
+                        <Field
+                          title={ariaLabel}
+                          ariaLabel={ariaLabel}
+                          key={idx}
```
Comment (aliaksei-chumakou) [inline src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js:44]: maybe `key` should be on the `FormattedMessage` now?

File: src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js [hunk 4]
```diff
@@ -56,15 +62,21 @@ const renderSubField = (name, config) => {
...
-      component={TextField}
-      disabled={config.disabled}
-      className={styles[`fixedFieldSubField${config.type}`]}
-      hasClearIcon={config.type !== SUBFIELD_TYPES.BYTE}
-      data-testid={`fixed-field-${config.type}`}
-    />
+    <FormattedMessage id="ui-quick-marc.record.subfield">
+      {ariaLabel => (
+        <Field
+          title={ariaLabel}
```
Comment (NikitaSedyx) [inline src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js:68]: titles are not required for fields

File: src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js [hunk 5]
```diff
@@ -36,15 +36,21 @@ const renderSubField = (name, config) => {
...
-                    <Field
-                      key={idx}
-                      name={`${fieldName}[${idx}]`}
-                      component={TextField}
-                      disabled={config.disabled}
-                      className={styles.fixedFieldSubFieldByte}
-                      hasClearIcon={false}
-                      data-testid={`fixed-field-${config.type}`}
-                    />
+                    <FormattedMessage id="ui-quick-marc.record.subfield">
```
Comment (NikitaSedyx) [inline src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js:39]: it's not subfield

File: src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js [hunk 6]
```diff
@@ -56,15 +62,21 @@ const renderSubField = (name, config) => {
...
-    <Field
-      name={fieldName}
-      label={label}
-      component={TextField}
-      disabled={config.disabled}
-      className={styles[`fixedFieldSubField${config.type}`]}
-      hasClearIcon={config.type !== SUBFIELD_TYPES.BYTE}
-      data-testid={`fixed-field-${config.type}`}
-    />
+    <FormattedMessage id="ui-quick-marc.record.subfield">
```
Comment (NikitaSedyx) [inline src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.js:65]: the same, not a subfield

**Pull request merged**